documentation/Windows-Installation-Guide.md

```diff
@@ -0,0 +1,399 @@
+- Click **Next** on the license page
+- Leave the installation directory to default on **Select Destination Location** box and click **Next**
+- Click **Next** on **Select Components** box
+- Click **Next** on **Select Start Menu Folder** box
+- Click **Next** on **Adjust your PATH environment** box 
+- Click **Next** on **Choosing HTTPS transport backend** box
+- Click **Next** on **Configuring the line ending conversions** box
+- Click **Next** on **Configuring the terminal emulator to use with Git Bash** box
+- Click **Install** on **Configuring extra options**, this will start installing Git on your machine
+- Once the installation is complete, click **Finish**
+
+###### Note: We still need to set environment variable in order for git command to work from Command Prompt
+
+### Set Environment Variable
+
+- Navigate to the **bin** directory of **Git** which in my case is ```C:\Program Files\Git\bin```. 
+Click on the address bar of the file explorer and copy the path.  
+![Set git env](https://user-images.githubusercontent.com/19670372/33494890-7ff4051e-d6bc-11e7-8bee-84b1ede0414c.png)
+- Navigate to the **Environment variable** window as described in the Java installation section, and when you are on 
+**Edit environment variable** screen, click on **New** and paste the address of the **Git bin directory path** that 
+you copied a little while ago and click **OK**  
+![Git env set](https://user-images.githubusercontent.com/19670372/33494887-7fb26898-d6bc-11e7-9abb-9565330b9ba1.png)
+- To test the installation again, open a new **Command Prompt** and type ```git --version``` 
+and you should see something like this.  
+![Git installed](https://user-images.githubusercontent.com/19670372/33494893-80291830-d6bc-11e7-9bff-9461fe5f4689.png)
+
+###### Congratulations, you now have git installed on your machine.
+
+##### Now that we have all the dependencies installed, let’s start working with Tombolo
+
+## Clone Tombolo Digital Connector Repository
+
+- To clone Tombolo Digital Connector repository, click [here](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector) 
+then click on **Clone or Download** and copy the path
+![Git tombolo clone](https://user-images.githubusercontent.com/19670372/33494907-8183c6b2-d6bc-11e7-8ad5-e743b46513ef.png)
+- Open a new **Command Prompt** and make sure that you present working directory is your home directory. 
+You can check that by looking the path in the **Command Prompt** window before **>**, in my case is looks like this
+![Check present working dir](https://user-images.githubusercontent.com/19670372/33494912-81f2adde-d6bc-11e7-9de5-0f40a097bba9.png)
+- If yours also follows the same pattern as ```C:\Users\<user-name>``` then you are good to go. If not then change 
+it by typing ```cd C:\Users\<user-name>```. Now that we are in our home directory, we can type following 
+commands to clone **TomboloDigitalConnector** repo.
+```
+cd Desktop
+mkdir Tombolo
```

Comment:
The user could skip this step as the clone will create the TomboloDigitalConnector directory.

Reply:
It is just to provide user a workspace for Tombolo project as a whole, that is why tombolo digital connector is a part of Tombolo. As we have web and python wrapper coming, it is just from the start giving them a workspace for tombolo project

Reply:
I see what you mean here.